scripts/sct_fmri_compute_tsnr.py

```diff
@@ -113,7 +113,7 @@ def main(args=None):
 
     arguments = parser.parse_args(args=None if sys.argv[1:] else ['--help'])
     fname_src = arguments.i
-    if "-o" in arguments:
+    if arguments.o is not None:
```

Comment:
Related to https://github.com/neuropoly/spinalcordtoolbox/issues/2926.
TODO rebase once that is merged.